src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -262,6 +251,14 @@ private async ValueTask<EvaluationEvent> EvaluateFeature<TContext>(string featur
                 FeatureDefinition = await GetFeatureDefinition(feature).ConfigureAwait(false)
             };
 
+            bool telemetryEnabled = evaluationEvent.FeatureDefinition.Telemetry != null && evaluationEvent.FeatureDefinition.Telemetry.Enabled;
```

Comment:
FeatureDefinition can be null here. This is the reason why the testcase failed. If you are referencing a feature flag which doesn't exist in the configuration, FeatureDefinition will be null.

Reply:
Thank you- I pulled that out to happen earlier and broke that test in the process. Updated.